docs/02.API-REFERENCE.md

```diff
@@ -3768,6 +3768,96 @@ jerry_value_to_bigint (const jerry_value_t value);
 - [jerry_value_is_bigint](#jerry_value_is_bigint)
 - [jerry_get_bigint_digits](#jerry_get_bigint_digits)
 
+## jerry_value_as_integer
+
+**Summary**
+
+Call the integer ecma builtin operation on the api value.
+
+**Prototype**
+
+```c
+jerry_value_t
+jerry_value_as_integer (const jerry_value_t value, double *number_p);
+```
+
+- `value` - api value
+- `number_p` - [out] double number
+- return value
+  - undefined value, if success
+  - thrown error, otherwise
+
+**Example**
+
+```c
+{
+  jerry_value_t number_val = jerry_create_number (123321);
+  double number;
+
+  number_val = jerry_value_as_integer (number_val, &number);
```

Comment:
Reassignment for `number_val` is misleading. I suggest : `jerry_value_t conv_result = jerry_value_as_integer (num_val, &number);`.  then release `num_val`